packages/react-components/tests/browser/RichTextInputBoxComponent.spec.tsx

```diff
@@ -112,3 +111,8 @@ const addList = async (listButtonLabel: string, component: Locator): Promise<voi
   editor.hover();
   await editor.pressSequentially('Third line');
 };
+
+const formatButtonClick = async (component: Locator): Promise<void> => {
```

Comment:
nit: this function could be pulled out to a utils file.

Reply:
Fixed in https://github.com/Azure/communication-ui-library/pull/5514